src/test/java/br/unb/cic/analysis/df/OverridingAssignmentAnalysisArraysConflictTest2.java

```diff
@@ -0,0 +1,64 @@
+package br.unb.cic.analysis.df;
+
+import br.unb.cic.analysis.AbstractMergeConflictDefinition;
+import br.unb.cic.analysis.SootWrapper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import soot.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class OverridingAssignmentAnalysisArraysConflictTest2 {
```

Comment:
would it make sense to add to the tests a link to the source code explored by each test?

Reply:
Could we post a comment with the code path? Some lines like: 31, 41 and 54 show this.